onnx/helper.py

```diff
@@ -905,3 +905,6 @@ def make_training_info(algorithm, algorithm_bindings, initialization, initializa
             binding.value = v
 
     return training_info
+
```

Comment:
nit: need 2 lines as spaces between two functions to fix flake8 issue. You can also check it in your local